presto-spi/src/main/java/com/facebook/presto/spi/block/ArrayBlockBuilder.java

```diff
@@ -24,6 +25,8 @@
         extends AbstractArrayBlock
         implements BlockBuilder
 {
+    private static final int INSTANCE_SIZE = ClassLayout.parseClass(ArrayBlockBuilder.class).instanceSize();
```

Comment:
We should include the size of `BlockBuilderStatus` (and its nested `PageBuilderStatus`) since both are required and pretty big.